### PR TITLE
Tempo TraceQl Editor update request #8382

### PR DESCRIPTION
**Pull request by jasuade**
Documentation update request for Tempo TraceQL Editor:
https://github.com/grafana/support-escalations/issues/8382

### CODE REVIEW

**Review by knylander-grafana — APPROVED** [review]
Thank you for updating the content!

File: docs/sources/shared/datasources/tempo-editor-traceql.md [hunk 1]
```diff
@@ -86,6 +86,8 @@ Query results for both the editor and the builder are returned in a table. Selec
 
 Selecting the trace ID from the returned results opens a trace diagram. Selecting a span from the returned results opens a trace diagram and reveals the relevant span in the trace diagram (above, the highlighted blue line).
 
+In the trace diagram, the bold text on the left side of each span indicates the service name, for example `mythical-requester: requester`, and it is hidden when subsequent spans have the same service name (nested spans). Each service has a color assigned to it, which is visible at the left of the name and timeline graph above. Spans with the same color belong to the same service. The grey text at the right side of the service name indicates the span name.
```
Comment (Eve832) [inline docs/sources/shared/datasources/tempo-editor-traceql.md:89]:
```suggestion
In the trace diagram, the bold text on the left side of each span indicates the service name, for example `mythical-requester: requester`, and it is hidden when subsequent spans have the same service name (nested spans). Each service has a color assigned to it, which is visible to the left of the name and timeline graph above. Spans with the same color belong to the same service. The grey text to the right of the service name indicates the span name.
```

File: docs/sources/shared/datasources/tempo-editor-traceql.md [hunk 2]
```diff
@@ -86,6 +86,8 @@ Query results for both the editor and the builder are returned in a table. Selec
 
 Selecting the trace ID from the returned results opens a trace diagram. Selecting a span from the returned results opens a trace diagram and reveals the relevant span in the trace diagram (above, the highlighted blue line).
 
+In the trace diagram, the bold text on the left side of each span indicates the service name, for example `mythical-requester: requester`, and it is hidden when subsequent spans have the same service name (nested spans). Each service has a color assigned to it, which is visible to the left of the name and timeline graph above. Spans with the same color belong to the same service. The grey text to the right of the service name indicates the span name.
```
Comment (Eve832) [inline docs/sources/shared/datasources/tempo-editor-traceql.md:89]:
```suggestion
In the trace diagram, the bold text on the left side of each span indicates the service name, for example `mythical-requester: requester`, and it is hidden when subsequent spans have the same service name (nested spans). Each service has a color assigned to it, which is visible to the left of the name and timeline in the graph in the image. Spans with the same color belong to the same service. The grey text to the right of the service name indicates the span name.
```

File: docs/sources/shared/datasources/tempo-editor-traceql.md [hunk 3]
```diff
@@ -86,6 +86,8 @@ Query results for both the editor and the builder are returned in a table. Selec
 
 Selecting the trace ID from the returned results opens a trace diagram. Selecting a span from the returned results opens a trace diagram and reveals the relevant span in the trace diagram (above, the highlighted blue line).
 
+In the trace diagram, the bold text on the left side of each span indicates the service name, for example `mythical-requester: requester`, and it is hidden when subsequent spans have the same service name (nested spans). Each service has a color assigned to it, which is visible to the left of the name and timeline in the graph in the image. Spans with the same color belong to the same service. The grey text to the right of the service name indicates the span name.
```
Comment (Eve832) [inline docs/sources/shared/datasources/tempo-editor-traceql.md:89]:
```suggestion
In the trace diagram, the bold text on the left side of each span indicates the service name, for example `mythical-requester: requester`, and it is hidden when subsequent spans have the same service name (nested spans). Each service has a color assigned to it, which is visible to the left of the name and timeline in the graph. Spans with the same color belong to the same service. The grey text to the right of the service name indicates the span name.
```

**Pull request merged**